### PR TITLE
Add relevant_areas percept handling

### PR DESCRIPTION
**Pull request by lisv**
This percept allows us to check for available land ready to be build on.

User Story: 
As a housing corporation agent I want to have access to all percepts from the environment and all indicators that will be useful to me so that I can function in the best way possible

### CODE REVIEW

**Comment by shdshdw** [comment]
This pull request contains conflicts, perhaps due to the recent merge of PR #27?

**Comment by JoshCode** [comment]
Approved, though please resolve the conflicts

**Comment by MHMCD** [comment]
looks good, can be merged if merge conflicts have been solved.

**Comment by lisv** [comment]
Conflicts are resolved now.

**Comment by MHMCD** [comment]
the conflicts weren't resolved correct.

**Comment by lisv** [comment]
I see. Let me try again

File: src/nl/codefox/contextvh/tygronEvents.mod2g [hunk 1]
```diff
@@ -65,6 +65,19 @@ module tygronEvents {
...
+	if percept(relevant_areas(CallID,MultiPolygons)), not(bel(relevant_areas(CallID,_)))
+		then insert(relevant_areas(CallID,MultiPolygons)).
+	% update relevant_areas/2
+	if percept(relevant_areas(CallID,NewMultiPolygons)), bel(relevant_areas(CallID,OldMultiPolygons))
+		then delete(relevant_areas(CallID,OldMultiPolygons))+ insert(relevant_areas(CallID,NewMultiPolygons)).
+		
+	%%% REQUEST PERCEPTS
+	% insert request/9 to beliefs
+	forall percept(request(Type,RequestType,RequestID,ContentLinkID,VisibieStakeholdersIDList,ActionLogIDList,Price,MultiPolygon,Area,Answers)) 
+		do insert(request(Type,RequestType,RequestID,ContentLinkID,VisibieStakeholdersIDList,ActionLogIDList,Price,MultiPolygon,Area,Answers)).
```
Comment (MHMCD) [inline src/nl/codefox/contextvh/tygronEvents.mod2g]: The `%%%REQUEST PERCEPT` can be removed completely, this is being handled by tygronRequests, if you remove line 77-80 then you are done.

Comment (lisv) [inline src/nl/codefox/contextvh/tygronEvents.mod2g]: They are removed now.

**Comment by JoshCode** [comment]
Why did you remove the request percepts?

**Comment by MHMCD** [comment]
https://github.com/CodeFoxNL/Contextproject-VH/pull/30#issuecomment-226241925 Because I told her to, it's being handled in the tygronrequests module

**Comment by JoshCode** [comment]
Is that how we want it though? It dilutes the lines of our previously clearly defined modules.

**Comment by lisv** [comment]
Since the request percept handling is fairly elaborate and important it has its own module.

**Comment by MHMCD** [comment]
The requests are a special case when it comes to perceiving, you want to directly handle some of the requests which have a single action and are of no use to the agent, they are confirmation for which an agent has no use, other requests are being added to the believe base.

**Comment by MHMCD** [comment]
Also I believe this pull request should not be too focussed on this problem, the module was added in #26 and in this pr these lines were removed from the tygron event module and you approved of this PR.

 Currently the pr is in a good state and I believe it should be merged soon.

**Comment by shdshdw** [comment]
With the issues resolved and with most members agreeing I think this can be merged as good reasoning has been provided for the issues @JoshCode mentioned.

**Comment by MatthijsJWH** [comment]
I approve of a merge

**Comment by JoshCode** [comment]
Alright, accepté.
